test/internal/cmd/generate-gateway-api-urls/main.go

```diff
@@ -12,26 +12,29 @@ import (
 	"text/template"
 )
 
-//go:generate go run . -crds-url $CRDS_URL -raw-repo-url $RAW_REPO_URL -in $INPUT -out $OUTPUT
+//go:generate go run . -crds-standard-url $CRDS_STANDARD_URL -crds-experimental-url $CRDS_EXPERIMENTAL_URL -raw-repo-url $RAW_REPO_URL -in $INPUT -out $OUTPUT
 
 var (
-	crdsURLFlag    = flag.String("crds-url", "", "The URL of Gateway API CRDs to be consumed by kustomize")
-	rawRepoURLFlag = flag.String("raw-repo-url", "", "The raw URL of Gateway API repository")
-	inFlag         = flag.String("in", "", "Template file path")
-	outFlag        = flag.String("out", "", "Output file path where the generate file will be placed")
+	crdsStandardURLFlag     = flag.String("crds-standard-url", "", "The URL of standard Gateway API CRDs to be consumed by kustomize")
+	crdsExperimentalURLFlag = flag.String("crds-experimental-url", "", "The URL of experimental Gateway API CRDs to be consumed by kustomize")
+	rawRepoURLFlag          = flag.String("raw-repo-url", "", "The raw URL of Gateway API repository")
+	inFlag                  = flag.String("in", "", "Template file path")
+	outFlag                 = flag.String("out", "", "Output file path where the generate file will be placed")
 )
 
 type Data struct {
-	CRDsKustomizeURL string
-	RawRepoURL       string
+	StandardCRDsKustomizeURL     string
+	ExperimentalCRDsKustomizeURL string
```

Comment:
Can we also rename these 2 to have `CRDs` prefixes? (similar to what has been done elsewhere)

Reply:
done.